services/user.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth import get_user_model
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
+    user = get_user_model().objects.create_user(username=username)
+    user.set_password(password)
```

Comment:
According to the checklist, you should call the `save()` method only once if possible. In the `create_user` function, you are calling `set_password` method which internally calls `save()`. It's better to set the password directly while creating the user. Here is an example: `user = get_user_model().objects.create_user(username=username, password=password)`.